librocksdb-sys/build.rs

```diff
@@ -45,6 +45,8 @@ fn bindgen_rocksdb() {
 }
 
 fn build_rocksdb() {
+    let target = env::var("TARGET").unwrap();
```

Comment:
If we don't set env variable "TARGET" we would get a panic. Is it really ok here ? What if get a target from `cfg!(target_arch)` by default, but if "TARGET" variable was set then get the value from it?